vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -43,18 +46,17 @@ class TextShadowNode final : public ShadowNodeBase {
   }
 
   void AddView(ShadowNode &child, int64_t index) override {
+    auto &childNode = static_cast<ShadowNodeBase &>(child);
+    VirtualTextShadowNode::ApplyTextTransformToNode(childNode, textTransform, /* hasChanged = */false);
+
+    auto run = childNode.GetView().try_as<winrt::Run>();
```

Comment:
I think this might be able to be a superclass of Run, either Inline or TextElement

Reply:
The point of checking if it's a Run is so we can take advantage of the TextBlock fast path and lift the Run content into the TextBlock.Text field: https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/text-block#performance-considerations